src/accordion/Accordion.test.js

```diff
@@ -0,0 +1,48 @@
+import test from 'ava';
+import jsdom from 'jsdom';
+
+import { Accordion } from './Accordion';
+
+test.beforeEach(t => {
+  global.document = jsdom.jsdom(`
+    <div id="accordion">
+      <ul class="accordion-list">
+        <li class="accordion-list-item">
+          <div class="accordion-section js-accordion-section">
+            <div class="accordion-section-toggle js-accordion-section-toggle">
+              <h4 class="accordion-section-toggle-content">Header</h4>
+            </div>
+            <ul class="accordion-section-list js-accordion-section-content">
+              <li class="accordion-section-list-item">
+                Content
+              </li>
+            </ul>
+          </div>
+        </li>
+        <li class="accordion-list-item">
+          <div class="accordion-section js-accordion-section">
+            <div class="accordion-section-toggle js-accordion-section-toggle">
+              <h4 class="accordion-section-toggle-content">Header</h4>
+            </div>
+            <ul class="accordion-section-list js-accordion-section-content">
+              <li class="accordion-section-list-item">
+                Content
+              </li>
+            </ul>
+          </div>
+        </li>
+      </ul>
+    </div>`);
```

Comment:
Can you import the template file? Copying output adds maintenance overhead and potential for out-of-sync.